account_invoice_supplierinfo_update/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_wizard_update_invoice_supplierinfo,access_wizard_update_invoice_supplierinfo,model_wizard_update_invoice_supplierinfo,base.group_user,1,0,0,0
```

Comment:
You must limit to the real users that access this wizard, and give write/create/unlink permissions to them